doc/en/weechat_plugin_api.en.adoc

```diff
@@ -15266,13 +15266,13 @@ Prototype:
 
 [source,c]
 ----
-struct t_gui_nick_group *weechat_nicklist_add_nick (struct t_gui_buffer *buffer,
```

Comment:
Could you please fix same issue in other languages?
Thanks!

Reply:
Ah, doh. Fixed now :)